questions/accessibility-aria.md

```diff
@@ -0,0 +1,36 @@
+### What is ARIA and when should you use it?
+
+#### Answer
+
+ARIA stands for "Accessible Rich Internet Applications". ARIA code is HTML. It provides additional attributes in the development of web applications to offer screen reader users more context about dynamic components, including the component's role, name, and state. It should only be used when an HTML element equivalent is not available or lacks full support. ARIA is semantic, and doesn't actually make your component dynamic.
+
+An example using ARIA:
+
+```
+<div 
+  role="combobox"
+  aria-expanded="false"
+  aria-owns="ex1-grid"
+  aria-haspopup="grid"
+  id="ex1-combobox">
+  ...
+</div>
+```
+Credit: W3C's [ARIA 1.1 Combobox with Grid Popup Example](https://w3c.github.io/aria-practices/examples/combobox/aria1.1pattern/grid-combo.html)
+
+#### Good to hear
+
+* Accessible Rich Internet Applications
+* Benefits screen reader users
```

Comment:
Benefits users with accessibility needs*

This includes:
- Motor
- Cognitive
- Visibility
- Hearing
- Alternative inputs

Reply:
That's why I stated screen reader "users", which can include more audience than just visually impaired. If it helps devs see a clearer picture, I'm all for clarity.

Reply:
Maybe...?
"Benefits screen readers users, such as people with visual and cognitive impairments"

Reply:
I would say `Benefits users with assistive technology needs` at the very least.

This includes switch users, screen-readers, and touch interface, and mobility, and cognitive.

Reply:
I've always been taught it was created for the benefit of screen reader users, but the spec does say assistive technology, so let's go with that.